plugins/modules/openafs_get_install_paths.py

```diff
@@ -259,8 +259,8 @@ def search_page(self, path, pattern):
         """
         Search an OpenAFS man page for a directory.
         """
-        with gzip.open(path) as z:
-            content = z.read().decode()
+        z = gzip.open(path)
```

Comment:
That is unfortunate. Since we don't have a context manager on that old (obsolete) version, now we will leak the file handle, which is not great.  I'll do a followup commit to close it.